LiteCore/Android/unicode/ndk_icu.c

```diff
@@ -169,7 +169,7 @@ UCollator* ucol_open(const char* loc, UErrorCode* status) {
   return ptr(loc, status);
 }
 
-void ucol_setAttribute(UCollator* coll, UColAttribute attr, UColAttributeValue value, UErrorCode* status) {
+void lc_ucol_setAttribute(UCollator* coll, UColAttribute attr, UColAttributeValue value, UErrorCode* status) {
```

Comment:
What's the reason for renaming all of these? (I assume `lc_` stands for LiteCore?)

Reply:
The ICU headers in the actual release define these function names as macros, so compilation was getting hairy with the existing names.  I wanted to make them distinct from that.